src/lib/tvgPictureImpl.h

```diff
@@ -169,10 +169,10 @@ struct Picture::Impl
 
     bool bounds(float* x, float* y, float* w, float* h) const
     {
-        if (x) *x = 0;
-        if (y) *y = 0;
-        if (w) *w = this->w;
-        if (h) *h = this->h;
+        if (x) *x = loader->vx;
```

Comment:
viewbox is only available to svg.
we can't assume these values are always valid.

I know... current bounds() api has an issue. 
This is also regarded to this issue. https://github.com/Samsung/thorvg/issues/746

Need to think about fundamental solution...